game/static/game/css/contributors.css

```
@@ -0,0 +1,573 @@
+.navbar {
+  position: sticky;
+  top: 0;
+  z-index: 1000;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 1.4rem 5%;
+
+  background: rgba(15, 15, 26, 0.9);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)